frame/utility/src/lib.rs

```diff
@@ -314,10 +318,22 @@ decl_module! {
 		/// - Storage: inserts one item, value size bounded by `MaxSignatories`, with a
 		///   deposit taken for its lifetime of
 		///   `MultisigDepositBase + threshold * MultisigDepositFactor`.
+		/// -------------------------------
+		/// - Base Weight:
+		///     - Create: 137.5 + 0.274 * S µs
+		///     - Approve: 103.8 + .266 * S µs
+		///     - Complete: 116.2 + .754 * S µs
+		/// - DB Weight:
+		///     - Reads: Multisig Storage, [Caller Account]
+		///     - Writes: Multisig Storage, [Caller Account]
+		/// - Plus Call Weight
 		/// # </weight>
 		#[weight = FunctionOf(
 			|args: (&u16, &Vec<T::AccountId>, &Option<Timepoint<T::BlockNumber>>, &Box<<T as Trait>::Call>)| {
-				args.3.get_dispatch_info().weight + 10_000 * (args.1.len() as Weight + 1)
+				args.3.get_dispatch_info().weight
+					.saturating_add(150_000_000 as Weight)
+					.saturating_add((500_000 as Weight).saturating_mul(args.1.len() as Weight))
```

Comment:
Please do not use `as` without good reason. The second `as` is of course necessary because there is no other infallible way to convert from a `usize`.